src/scss/components/topnav/mixins/_topNavControl.scss

```diff
@@ -8,16 +8,16 @@
     box-shadow: $focus-glow;
 
     &::-ms-check {
-      border-color: $cyan-700;
+      border-color: var(--hxTopNav-focus-ie-backgroundColor, $cyan-700);
```

Comment:
Please update to the Color Palette v2.0 color variables.

Reply:
Changes have been done, please review.